autoload/lsp/ui/vim.vim

```diff
@@ -292,6 +484,21 @@ function! lsp#ui#vim#code_action() abort
     echo 'Retrieving code actions ...'
 endfunction
 
+function! s:jump_to_location(loc) abort
+    let l:idx = split(a:loc, '\.')[0]
+    execute 'cc' l:idx
+endfunction
+
+function! s:show_locations() abort
+    if !get(g:, 'lsp_fzf_enable') || !get(g:, 'loaded_fzf')
+        botright copen
+    else
+        let l:qfl = getqflist()
+        call map(l:qfl, {idx, item -> string(idx + 1) . '. ' . fnamemodify(expand(bufname(item['bufnr'])), ":~:.") . ':' . string(item['lnum']) . ':' . string(item['col']) . ':' . item['text']})
+        call fzf#run(fzf#wrap({'source': l:qfl, 'sink': function('s:jump_to_location'), 'options': '--reverse +m --prompt="Jump> "'}))
+    endif
```

Comment:
Please reverse the condition. We may add yet-another selector in future.
```
if extensibie1
  call do_extend1()
  return
endif
if extensibie2
  call do_extend2()
  return
endif
call do_original()
```

Reply:
Yes, that makes sense. Fixed.